src/core/unicode.js

```diff
@@ -1630,8 +1630,29 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function keepUnifiedIdeograph(str, normalizedUnicodesMap) {
```

Comment:
The function's name is form https://en.wikipedia.org/wiki/CJK_Unified_Ideographs. 
You can see the https://unicode.org/charts/PDF/UF900.pdf.
I think it is a official name.

Reply:
I think that the comment still applies, since the new function is doing much more than *just* handling the ideographs (i.e. the str.length === 1 case).

Reply:
1. `str.length===1`, replace it  with unified unicode or return original text
2. `str.length>2`, keep the unified unicode
What goes wrong here? Can you explain that in detail?

Reply:
The name feels weird, given that *all* toUnicode values regardless if they're Unified Ideographs or not are passed through this function.

Reply:
Apart from the function name temporarily.
If don't pass the toUnicode values to a function, how to judge whether it is Unified Ideograph?
Are you talking about the performance problem? The performance is almost the same as original code. Both are O(N). N is the number of the toUnicode value.

Reply:
This isn't about performance, it's about a confusing *name* of new helper function.

Reply:
I know what's your meaning. I will change it to normalizeUnicode